### PR TITLE
#162: Fix overlap in tool styling

### LINKED ISSUE #162: Footer styling on tools pages
On tools pages (e.g., chemprops) and the data upload page, the footer overlaps with the body of the page

### PR DESCRIPTION
**Pull request by aswallace**
The overlap seems to be from the `md-layout-item` height parameter, not from the footer, but not sure if this was the best possible fix here. Went for the option that seemed to minimize effect on existing files (avoiding a full refactor for now)

fix #162

### CODE REVIEW

File: nginx/default.conf [hunk 1]
```diff
@@ -39,9 +39,10 @@ server {
     send_timeout 1500;
   }
 
-  location /whyis {
-    rewrite /whyis/(.*) /$1 break;
-    proxy_pass http://whyis;
+  # location /whyis {
+  #   rewrite /whyis/(.*) /$1 break;
+  #   proxy_pass http://whyis;
+  # }
```
Comment (stouffers) [inline nginx/default.conf:45]: Are these changes intended for this ticket?

**Review by stouffers — APPROVED** [review]
Works and looks good. Not sure why nginx config is modified though.

File: nginx/default.conf (hunk 1 above)
Comment (aswallace) [reply nginx/default.conf:45]: Similar to the other one -- I added the closing curly brace in line 45, but have commented for now since it seems like there might also be other things missing addressed in PR #169
